### ISSUE TITLE
issue2

### ISSUE THREAD

**Issue by zyy1234q** [Jiang-HB/VBReg#4]
I noticed that you use 0.03cm voxels to downsample, and your comparison method is 0.05cm voxels downsample, is this an unfair comparison?

**Comment by Jiang-HB** [comment]
We also use voxel-size with 0.05m to pre-process the data. Please refer to the misc/cal_fcgf.py and misc/cal_fpfh.py for more details.

**Comment by zyy1234q** [comment]
Is it reasonable to use the pre-trained model you provided to run an RR of 92.81?Since I only have 24G of video memory, so numnode=5000, instead of 'all', this effect will not go from 93.53->92.81.

**Comment by Jiang-HB** [comment]
Is it possible to increase the number of nodes from 5000 to 8000 or 10000? A smaller number of nodes could indeed degrade the final performance.

**Comment by zyy1234q** [comment]
your video memory has more 24G？

---- Replied Message ----
| From | ***@***.***> |
| Date | 07/09/2024 11:20 |
| To | Jiang-HB/VBReg ***@***.***> |
| Cc | zyy1234q ***@***.***>,
Author ***@***.***> |
| Subject | Re: [Jiang-HB/VBReg] issue2 (Issue #4) |

Is it possible to increase the number of nodes from 5000 to 8000 or 10000? A smaller number of nodes could indeed degrade the final performance.

**Comment by Jiang-HB** [comment]
I tested the model on both the V100 (32GB) and the TITAN RTX (24GB). Both tests were successful without encountering any ‘out of memory’ errors. Therefore, it’s advisable to increase the numnode value as much as possible on your machine with 24G memory. The numnode of 5000 is too small.

**Comment by zyy1234q** [comment]
all used “all”?

---- Replied Message ----
| From | ***@***.***> |
| Date | 07/09/2024 11:31 |
| To | Jiang-HB/VBReg ***@***.***> |
| Cc | zyy1234q ***@***.***>,
Author ***@***.***> |
| Subject | Re: [Jiang-HB/VBReg] issue2 (Issue #4) |

I tested the model on both the V100 (32GB) and the TITAN RTX (24GB). Both tests were successful without encountering any ‘out of memory’ errors. Therefore, it’s advisable to increase the numnode value as much as possible on your machine with 24G memory. The numnode of 5000 is too small.

**Comment by Jiang-HB** [comment]
Yes, I suggest adding an if-else statement at Line 40 in "test_3DMatch.py" to prevent this error. For example, if numnode > 10000 (determined by yourself), you can clip them to 10000 using src_keypts[:, :10000]. As such, you can constrain the scale of too large input.

**Comment by zyy1234q** [comment]
OK，thansks

---- Replied Message ----
| From | ***@***.***> |
| Date | 07/09/2024 11:47 |
| To | Jiang-HB/VBReg ***@***.***> |
| Cc | zyy1234q ***@***.***>,
Author ***@***.***> |
| Subject | Re: [Jiang-HB/VBReg] issue2 (Issue #4) |

Yes, I suggest adding an if-else statement at Line 40 in "test_3DMatch.py" to prevent this error. For example, if numnode > 10000 (determined by yourself), you can clip them to 10000 using src_keypts[:, :10000]. As such, you can constrain the scale of too large input.

**Comment by zyy1234q** [comment]
Hello author, what do the horizontal and vertical coordinates in the diagram mean, and then, if I want to draw such a diagram how it should work
